### ISSUE TITLE
Can't setCameraMode to DJICameraModePlayback

### ISSUE THREAD

**Issue by maciekish** [dji-sdk/Mobile-SDK-iOS#69]
I can't set my camera to playback mode. Code -1005 "The parameters are invalid" both in my own project and the official panorama demo.

Steps to reproduce:

Download panorama demo, run, connect and tap download https://github.com/DJI-Mobile-SDK-Tutorials/iOS-PanoramaDemo

You need to set a breakpoint to catch the error as it is not handled.

-  iOS DJI SDK 3.5.1
-  Phantom 4 Pro
-  iOS 10.2
-  iPhone 7 Plus
-  Xcode 8.2.1

Forum post http://forum.dev.dji.com/forum.php?mod=viewthread&tid=32995&page=1&extra=#pid228898

**Comment by oliverou** [comment]
Hi @maciekish, Phantom 4 Pro doesn't support `DJICameraModePlayback` mode, please use `DJICameraModeMediaDownload` instead.

**Comment by maciekish** [comment]
Yeah i figured as much eventually. Why does the demo project use the `DJICameraModePlayback` select-dance instead of simply listing the media and downloading it directly?

**Comment by oliverou** [comment]
Hi @maciekish, I have updated the README file of the PanoramaDemo Github Repo with **Not Supported DJI Products**. The demo is a demonstration sample, if you want to support Phantom 4 Pro, you can modify it and use `DJICameraModeMediaDownload ` instead.

**Comment by hy9be** [comment]
@oliverou It seems I cannot use the MultiplePreviewMode with media download as the camera mode: 
![image](https://user-images.githubusercontent.com/3392907/28390413-fcec1112-6ca7-11e7-849d-ebfca8a8401a.png)

Is that right?

**Comment by XietongLU** [comment]
You are right. In media download mode, you should only use interface in DJIMediaManager.
